vsts/pipelines/ci-detector.yml

```diff
@@ -0,0 +1,94 @@
+resources:
+- repo: self
+
+variables:
+- group: Oryx
+
+stages:
+  - stage: ValidatePipeline
+    displayName: Validate Pipeline
+    jobs:
+    - job: ValidatePipeline
+      displayName: Validate pipeline
+      pool:
+        name: OryxLinux
+      variables:
+        skipComponentGovernanceDetection: true
+      steps:
+      - script: |
+          set -x
+          if [ "$(Build.Reason)" != "Manual" ]; then
+            echo
+            echo "Invalid configuration."
+            echo "Build should be Manual"
+            exit 1
+          fi
+          if [ "$(Build.DefinitionName)" != "Oryx-Detector-CI" ]; then
```

Comment:
why do we need this check?

Reply:
(I followed the pattern that we have in existing Oryx-CI). I think this was a gate added to make sure that only this build definition can make release builds which also ensures that we only release signed bits.